docs/Employer/4 - Individual.md

```diff
@@ -42,29 +42,29 @@ Parameter | Type | Required | Description
       "code": 200,
       "body": {
         "id": "5d0b10a1-a09a-430f-81f1-20be735dc5e9",
-        "first_name": "Jeremy",
+        "first_name": "Jane",
         "middle_name": null,
-        "last_name": "Zhang",
+        "last_name": "Doe",
         "emails": [
           {
-            "data": "jeremy@tryfinch.com",
+            "data": "janedoe@tryfinch.com",
             "type": "work"
           },
           {
-            "data": "jeremyziyuzhang@gmail.com",
+            "data": "janedoe@gmail.com",
             "type": "personal"
           }
         ],
         "phone_numbers": [
           {
-            "data": "4077313626",
+            "data": "4151234567",
```

Comment:
`xxx55501xx`
see: https://softwareengineering.stackexchange.com/questions/376535/whats-the-phone-number-equivalent-of-example-org